src/graph_compute/phase_scheduling.h

```diff
@@ -40,6 +40,8 @@ static inline bool interChunkDependency(vid_t v, vid_t w) {
   static const vid_t chunkMask = (1 << CHUNK_BITS) - 1;
   if ((v >> CHUNK_BITS) == (w >> CHUNK_BITS)) {
     return false;
+  } else if ((v & chunkMask) == (w & chunkMask)) {
```

Comment:
Oh, that's weird - I haven't touched this code in quite a while.  I'm not sure why master had this bug.  It looks like it got fixed when I committed the base meshes, so I must have had that bug fix in my local copy.